src/components/common/DynamicTabs/DynamicTabs.scss

```diff
@@ -0,0 +1,196 @@
+.dynamic-tabs-section {
+    .dynamic-tabs-container {
+        border-left: 1px solid var(--N200);
+        border-right: 1px solid var(--N200);
+        overflow-x: auto;
+        -ms-overflow-style: none;
+        scrollbar-width: none;
+
+        &::-webkit-scrollbar {
+            display: none;
+        }
+    }
+
+    .more-tabs-wrapper {
+        .more-tabs-option {
+            margin-left: 8px;
+            box-sizing: border-box;
+            display: flex;
+            align-items: center;
+            padding: 2px;
```

Comment:
Can we minimize css using formulae classes?

Reply:
Done